azure-pipelines.yml

```diff
@@ -144,6 +144,14 @@ jobs:
       continueOnError: true
       condition: not(succeeded())
 
+    - task: PublishBuildArtifacts@1
+      displayName: NuPkg
```

Comment:
NuPkg isn't very descriptive. Consider naming this `Publish Artifact Packages`

Reply:
💡 

```suggestion
     displayName: Publish NuGet Packages
```

Reply:
Publish NuGet packages is not ideal - that is easily confused with publish to myget/nuget.